assets/images/down.svg

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 24.3.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 20 20" style="enable-background:new 0 0 20 20;" xml:space="preserve">
+<path d="M 17.942 7.325 L 11.342 13.925 C 10.542 14.725 9.442 14.725 8.742 13.925 L 2.142 7.325 C 1.942 7.125 1.942 6.725 2.142 6.425 L 2.942 5.625 C 3.142 5.425 3.542 5.425 3.842 5.625 L 9.942 11.725 L 16.042 5.625 C 16.242 5.425 16.642 5.425 16.942 5.625 L 17.842 6.525 C 18.042 6.725 18.042 7.125 17.942 7.325 Z"/>
+</svg>
```

Comment:
annoyingly I couldn't pass a transform property into the SVG component as it crashed the RNSVG viewer on android. So I had to edit the existing SVG and rotate it

Reply:
This seems fine. I'm not sure using a transform would be worth the added complexity anyway.